Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1005,6 +1005,19 @@ class ISiteSchema(Interface):
         required=False,
     )
 
+    site_favicon = schema.Bytes(
+        title=_('Site Favicon'),
+        description=_('This shows a custom favicon on your site.'),
+        required=False,
+    )
+    
+    site_favicon_mimetype = schema.TextLine(
+        title=_('Site Favicon Mimetype'),
+        description=_('Automatically setted Mimetype of the Favicon'),
```

Comment:
Change to "MIME type of the favicon (automatically set when a new favicon is uploaded)".